django_graph_api/tests/test_introspection.py

```diff
@@ -1,14 +1,268 @@
+        ('episode', schema.query_root._declared_fields['episode']),
+        ('episodes', schema.query_root._declared_fields['episodes']),
+        ('hero', schema.query_root._declared_fields['hero']),
+    ]
+
+
+def test_type__object__get_inputFields():
+    type_object = TypeObject(None, TypeObject, None)
+    assert type_object.get_inputFields() is None
+
+
+def test_type__object__get_interfaces():
+    type_object = TypeObject(None, TypeObject, None)
+    assert type_object.get_interfaces() == []
+
+
+def test_type__object__get_possibleTypes():
+    type_object = TypeObject(None, TypeObject, None)
+    assert type_object.get_possibleTypes() is None
+
+
+def test_type__object__get_enumValues():
+    type_object = TypeObject(None, TypeObject, None)
+    assert type_object.get_enumValues() is None
+
+
+def test_type__object__get_ofType():
+    type_object = TypeObject(None, TypeObject, None)
+    assert type_object.get_ofType() is None
+
+
+def test_type__enum__get_fields():
+    type_object = TypeObject(None, TypeKindEnum, None)
+    assert type_object.get_fields() is None
+
+
+def test_type__enum__get_inputFields():
+    type_object = TypeObject(None, TypeKindEnum, None)
+    assert type_object.get_inputFields() is None
+
+
+def test_type__enum__get_interfaces():
+    type_object = TypeObject(None, TypeKindEnum, None)
+    assert type_object.get_interfaces() is None
+
+
+def test_type__enum__get_possibleTypes():
+    type_object = TypeObject(None, TypeKindEnum, None)
+    assert type_object.get_possibleTypes() is None
+
+
+def test_type__enum__get_enumValues():
+    type_object = TypeObject(None, TypeKindEnum, None)
+    assert type_object.get_enumValues() == TypeKindEnum.values
+
+
+def test_type__enum__get_ofType():
+    type_object = TypeObject(None, TypeKindEnum, None)
+    assert type_object.get_ofType() is None
+
+
+def test_type__list__get_name():
+    type_object = TypeObject(None, List(Character), None)
+    assert type_object.get_name() is None
+
+
+def test_type__list__get_fields():
+    type_object = TypeObject(None, List(Character), None)
+    assert type_object.get_fields() is None
+
+
+def test_type__list__get_inputFields():
+    type_object = TypeObject(None, List(Character), None)
+    assert type_object.get_inputFields() is None
+
+
+def test_type__list__get_interfaces():
+    type_object = TypeObject(None, List(Character), None)
+    assert type_object.get_interfaces() is None
+
+
+def test_type__list__get_possibleTypes():
+    type_object = TypeObject(None, List(Character), None)
+    assert type_object.get_possibleTypes() is None
+
+
+def test_type__list__get_enumValues():
+    type_object = TypeObject(None, List(Character), None)
+    assert type_object.get_enumValues() is None
+
+
+def test_type__list__get_ofType():
+    type_object = TypeObject(None, List(Character), None)
+    assert type_object.get_ofType() is Character
+
+
+def test_field__get_type():
+    field_object = FieldObject(None, (
```

Comment:
I find it easier to read on one line, but this is not a big deal.

Reply:
I've tweaked this... think it's better now.